packages/api-svc/src/routes.ts

```diff
@@ -339,6 +339,10 @@ export class ExpressRoutes {
 		let currencyCodesStr = req.query.currencyCodes as string;
 		let batchStatusesStr = req.query.batchStatuses as string;
 
+		// Pagination
```

Comment:
We need this to accept both cases, so pageIndex and pageindex are the te same, use somehting like this (from transfers):


```
// optional pagination
            const pageIndexStr = req.query.pageIndex as string || req.query.pageindex as string;
            const pageIndex = pageIndexStr ? parseInt(pageIndexStr) : undefined;

            const pageSizeStr = req.query.pageSize as string || req.query.pagesize as string;
            const pageSize = pageSizeStr ? parseInt(pageSizeStr) : undefined;
```